data/hai/hai missions.txt

```diff
@@ -298,15 +298,37 @@ mission "Returning Home"
 			`You are stopped by a human in the <origin> marketplace. He takes off his hat and asks, "You're from the outside, aren't you?"`
 			choice
 				`	"Yes, I am."`
+					to display
+						not "hai space start"
 					goto outsider
+				`	"Not really. I grew up here."`
+					to display
+						has "hai space start"
+					goto clothe
 				`	"What do you mean 'the outside?'"`
 			
 			`	"Outside of Hai space. You don't look quite like most humans who live here, unless there's some new fashion trend going on that I don't know about."`
 			choice
 				`	"Oh, yes. I do come from outside."`
+					to display
+						not "hai space start"
+				`	"Not really. I grew up here."`
+					to display
+						has "hai space start"
+					goto outsider
 			
+			label clothe
+			`	"That's strange. You don't look like most do here," he says.`
 			label outsider
+			`	You explain to him that these clothes used to belong to your father, who was a merchant from human space; your mother kept some for you when he left.`
+				to display
+					has "hai space start"
 			`	"Good. My name is Elliot." You introduce yourself to Elliot and shake hands.`
+				to display
+					has "human space start"
+			`	He sighs. "I was hoping to get someone from human space. Well, a captain's a captain, I guess. My name is Elliot." You introduce yourself to Elliot and shake hands.`
+				to display
+					not "human space start"
```

Comment:
This is a clumsy way of handling this mission, and it makes some pretty big assumptions about the player. (Have I really not bought any clothes the entire time I've been a captain?) The clothing thing isn't that vital to the mission, either; Arthur and Kiru don't approach you just because you're a captain from the outside.
```suggestion
			`You are stopped by a human in the <origin> marketplace. He takes off his hat and asks, "You're from the outside, aren't you?"`
			choice
				`	"Yes, I am."`
					to display
						not "hai space start"
					goto outsider
				`	"Not really. I grew up here."`
					to display
						has "hai space start"
					goto clothe
				`	"What do you mean 'the outside?'"`
			
			`	"Outside of Hai space. You don't look quite like most humans who live here, unless there's some new fashion trend going on that I don't know about."`
			choice
				`	"Oh, yes. I do come from outside."`
					to display
						not "hai space start"
				`	"Not really. I grew up here."`
					to display
						has "hai space start"
					goto outsider
			
			label clothe
			`	"That's strange. You don't look like most do here," he says.`
			label outsider
			`	You explain to him that these clothes used to belong to your father, who was a merchant from human space; your mother kept some for you when he left.`
				to display
					has "hai space start"
			`	"Good. My name is Elliot." You introduce yourself to Elliot and shake hands.`
				to display
					has "human space start"
			`	He sighs. "I was hoping to get someone from human space. Well, a captain's a captain, I guess. My name is Elliot." You introduce yourself to Elliot and shake hands.`
				to display
					not "human space start"
```